templates/rhel9.tpl.yaml

```diff
@@ -3,7 +3,7 @@ kind: Template
 metadata:
   name: {{ os }}-{{ item.workload }}-{{ item.flavor }}
   annotations:
-    openshift.io/display-name: "Red Hat Enterprise Linux 9.0 VM"
+    openshift.io/display-name: "Red Hat Enterprise Linux 9 VM"
```

Comment:
I like this idea, but for consistency reasons this should be changed also for the RHEL 6, 7 and 8 template.